bot/cogs/fun.py

```diff
@@ -171,23 +171,32 @@ def __init__(self, bot: Bot):
         A #quotes channel must be set using the set_quote_channel command in order for this command to work.
         """
         if self.quote_channel is None:
-            await ctx.send("Please set the quotes channel.")
-            return
+            self.quote_channel = self.bot.get_channel(QUOTES_CHANNEL_ID)
         quotation_channel = self.quote_channel
         if member is not None:
-            all_quotations = await quotation_channel.history(limit=None).flatten()
             quotations = []
-            for quotation in all_quotations:
-                embed = quotation.embeds[0]
-                author_name = embed.author.name
-                author = ctx.message.guild.get_member_named(author_name)
-                if author == member:
+            async for quotation in quotation_channel.history(limit=None):
```

Comment:
Should result in a more responsive bot, but it needs to be tested first.

Reply:
This would cause the bot to iterate every message sent in the quotes channel if the user isn't found. Have you tested the performance implications of this?